animations/pubspec.yaml

```diff
@@ -0,0 +1,18 @@
+name: animations
+description: A new Flutter project.
+version: 1.0.0+1
+
+environment:
+  sdk: ">=2.3.0 <3.0.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+  cupertino_icons: ^0.1.2
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+flutter:
+  uses-material-design: true
+  assets:
+    - assets/
```

Comment:
Where did the wolf and cat images come from? Do we hold the copyright for them?

Reply:
I got them both from Pixabay:

https://pixabay.com/photos/wolf-head-profile-predator-2878633/
https://pixabay.com/photos/cat-kitty-pet-feline-animal-4269479/

Reply:
While you're right that we could probably use those without issue, it'd be better to use something we know 100%, no-doubt-about-it belongs to us. I've emailed you a link to a Drive folder with images to which we own the copyright.